README.md

```diff
@@ -1,47 +1,93 @@
 # PocketSVG
 
-A simple toolkit for using displaying and manipulating SVGs on iOS and OS X.
+[![Build Status](https://travis-ci.org/arielelkin/PocketSVG.svg?branch=master)](https://travis-ci.org/arielelkin/PocketSVG)
+
+A simple toolkit for displaying and manipulating SVGs on iOS and macOS.
 
 Convert your SVGs into 
```

Comment:
This section seems redundant. (plus, we're not really turning an SVG into a UI/NSView/Layer, we do load it as a UI/NSBezierPath (not NSBezierCurve :)